### ISSUE TITLE
Training hyperparmeter for CheXpert dataset

### ISSUE THREAD

**Issue by hankyul2** [Optimization-AI/LibAUC#20]
Hi, Thank you for sharing your great work.

I am wondering if you can provide some missing hyperparameter for training CheXpert dataset. I summarize training hyperparameters explained in 4.2.1. of [DAM paper](https://arxiv.org/pdf/2012.03173.pdf) and denote missing values with `?` in below table. If you can provide these missing values, I would appreciate it very much.

CrossEntropy optimization
| setting      | value                                                        |
| ------------ | ------------------------------------------------------------ |
| augmentation | RandomAffine(rotation=`?`, translation=`?`, scaling=`?`) |
| criterion    | CE                                                           |
| optimizer    | Adam or AdamW (`?`)                                                        |
| epoch        | 2                                                            |
| batch        | 32                                                           |
| lr           | 1e-5                                                         |
| w.d.         | 1e-5                                                         |

AUC Maximization
| setting           | value                                                        |
| ----------------- | ------------------------------------------------------------ |
| augmentation | RandomAffine(rotation=`?`, translation=`?`, scaling=`?`) |
| criterion         | AUCM_MultiLabel                                              |
| optimizer         | PESG                                                         |
| epoch             | 2                                                            |
| batch             | 32                                                           |
| lr                | 0.1                                                          |
| scheduler | decrease 3 times at 2000, 8000 iter (`?`) |
| w.d.              | 0                                                            |
| gamma   | choose between (300, 500, 800) (`?`) |

Thank you again!

**Comment by hankyul2** [comment]
Hi! I am sorry to ask you without reading your code thoroughly. 
The augmentation is already applied in dataset shown in below snippets. ([link](https://github.com/Optimization-AI/ICCV2021_DeepAUC/blob/3193440baf29cf1e9ff78fc32544fa9e2a996d40/chexpert.py#L143))

```python
def image_augmentation(self, image):
    img_aug = tfs.Compose([tfs.RandomAffine(degrees=(-15, 15), translate=(0.05, 0.05), scale=(0.95, 1.05), fill=128)]) # pytorch 3.7: fillcolor --> fill
    image = img_aug(image)
    return image
```

Thank you
